jenkins/deploy.sh

```diff
@@ -96,6 +102,14 @@ echo "Deploy CMD: $DEPLOY_CMD"
 
 ###### Deploy the parent pom file ######
 $DEPLOY_CMD -Dfile=./pom.xml -DpomFile=./pom.xml
+PARENT_ART_ID=$(mvnEval "./" project.artifactId)
+echo "$ART_GROUP_ID:$PARENT_ART_ID:$ART_VER:pom" >> $ARTIFACT_FILE
+
+###### Deploy the jdk-profile pom file ######
```

Comment:
Deploy pom.xml file of the module `jdk-profiles` here, to PASS the release artifact's dependency check